### ISSUE TITLE
java.lang.NoSuchMethodException: startActionMode in ActionBarCompat

### ISSUE THREAD

**Issue by fedefernandez** [ManuelPeinado/MultiChoiceAdapter#17]
Hi,

There is a bug with ActionBarCompat. In com.manuelpeinado.multichoiceadapter.extras.actionbarcompat.MultiChoiceAdapterHelper you need to call "startSupportActionMode" method in startActionMode.

I have tested with this code and is working fine now.

```
@Override
protected void startActionMode() {
    try {
        Activity activity = (Activity) adapterView.getContext();
        Method method = activity.getClass().getMethod("startSupportActionMode", ActionMode.Callback.class);
        actionMode = (ActionMode) method.invoke(activity, owner);
    } catch (Exception e) {
        e.printStackTrace();
        throw new RuntimeException(e);
    }
}
```

Thanks for the library.

**Comment by ManuelPeinado** [comment]
Thanks a lot for reporting this, I'll upload a fixed version of the lib
ASAP.

**Comment by ManuelPeinado** [comment]
Fixed this in v3.0.1, which is on its way to maven central.

**Comment by fedefernandez** [comment]
Fixed, thanks!
